src/geometry/geom2/toSides.js

```diff
@@ -9,7 +9,7 @@ const canonicalize = require('./canonicalize')
  * let sharedsides = toSides(geometry)
  */
 const toSides = function (geometry) {
```

Comment:
ahh smart !

Reply:
I really like this, makes sides data independant from data structures, well done